packages/fxa-auth-server/docs/swagger/emails-api.ts

```diff
@@ -120,11 +120,11 @@ const RECOVERY_EMAIL_POST = {
         400: {
           description: dedent`
             Failing requests may be caused by the following errors (this is not an exhaustive list):
-            \`errno: 104\` - Unverified account
-            \`errno: 138\` - Unverified session
-            \`errno: 139\` - Can not add secondary email that is same as your primary
-            \`errno: 140\` - Email already exists
-            \`errno: 141\` - Email already exists
+            - \`errno: 104\` - Unverified account
+            - \`errno: 138\` - Unverified session
+            - \`errno: 139\` - Can not add secondary email that is same as your primary
+            - \`errno: 140\` - Email already exists
+            - \`errno: 141\` - Email already exists
```

Comment:
Can you confirm these two errors?
Feels fishy that we have two error codes for the same message. 🐟

Reply:
Yep, they're the same.

https://github.com/mozilla/fxa/blob/e9e80718641592aef71bb9368031cc4cf913f1c8/packages/fxa-auth-server/docs/api.md#post-recovery_email